src/sparsezoo/models/zoo.py

```diff
@@ -20,13 +20,20 @@
 from typing import List, Union
 
 from sparsezoo.objects.model import Model
-from sparsezoo.objects.optimization_recipe import OptimizationRecipe
+from sparsezoo.objects.optimization_recipe import (
+    OptimizationRecipe,
+    OptimizationRecipeTypes,
+)
 from sparsezoo.requests import ModelArgs, download_get_request, search_get_request
 
 
 __all__ = ["Zoo"]
 
 
+# optional prefix for stubs
+_ZOO_STUB_PREFIX = "zoo:"
```

Comment:
Small thing, can we declare this as public and export it so other integrations can use it explicitly?